pkgs/tools/wayland/wayland-proxy-virtwl/default.nix

```
@@ -20,20 +23,22 @@ ocamlPackages.buildDunePackage rec {
   '';
 
   useDune2 = true;
-  minimumOCamlVersion = "4.08";
+  minimumOCamlVersion = "4.12";
```

```suggestion
  minimalOCamlVersion = "4.12";
```

(suggestion applies to the other package too)

Also please drop the `useDune2` attributes.

---

Done!